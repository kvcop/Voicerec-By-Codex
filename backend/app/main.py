"""Main application entry point."""

from fastapi import FastAPI

app = FastAPI(title='Meeting Recognizer')


@app.get('/health')
def health_check() -> dict[str, str]:
    """Return service health status."""
    return {'status': 'ok'}
