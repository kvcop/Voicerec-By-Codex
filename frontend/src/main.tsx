import React from 'react';
import ReactDOM from 'react-dom/client';

export function App() {
  return <h1>Voicerec</h1>;
}

const rootElement = typeof document === 'undefined' ? null : document.getElementById('root');
if (rootElement) {
  ReactDOM.createRoot(rootElement).render(
    <React.StrictMode>
      <App />
    </React.StrictMode>
  );
}

export default App;
