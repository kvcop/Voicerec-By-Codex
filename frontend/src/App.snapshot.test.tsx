import { describe, it, expect, beforeEach } from 'vitest';
import { render } from '@testing-library/react';
import { Root } from './main';

describe('App snapshot', () => {
  beforeEach(() => {
    localStorage.clear();
  });

  it('matches the initial render snapshot', () => {
    const { container } = render(<Root />);
    expect(container).toMatchSnapshot();
  });
});
