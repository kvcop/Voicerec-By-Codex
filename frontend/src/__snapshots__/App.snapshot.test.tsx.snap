// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App snapshot > matches the initial render snapshot 1`] = `
<div>
  <button
    data-testid="switch"
  >
    RU
  </button>
  <div
    class="_container_f2db1c"
  >
    <h1
      class="_title_f2db1c"
    >
      Voicerec
    </h1>
    <div
      class="_content_f2db1c"
    >
      <button
        aria-controls="radix-:r0:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="_trigger_80b6b1"
        data-state="closed"
        type="button"
      >
        Open Dialog
      </button>
      <section
        aria-live="polite"
        class="_container_15c5d8"
      >
        <div
          class="_header_15c5d8"
        >
          <h2
            class="_title_15c5d8"
          >
            Live transcript
          </h2>
          <span
            class="_status_15c5d8"
            data-state="unsupported"
          >
            Live transcript is unavailable in this environment.
          </span>
        </div>
        <p
          class="_placeholder_15c5d8"
        >
          Live transcript is unavailable in this environment.
        </p>
      </section>
    </div>
  </div>
</div>
`;
